src/hooks/useAddTokenToMetamask.ts

```diff
@@ -30,7 +30,10 @@ export default function useAddTokenToMetamask(token: Token | undefined): {
               address: token.addresses[chainId],
               symbol: token.symbol,
               decimals: token.decimals,
-              image: getTokenIconPath(token.symbol),
+              image:
+                token.symbol === "SDL"
+                  ? "https://assets.coingecko.com/coins/images/20476/small/edit_saddle.png?1660734429"
+                  : getTokenIconPath(token.symbol),
```

Comment:
LGTM!

This logo is better than the logo of saddle token on sushiswap.
For some reason sushi swap saddle logo is having dark border.